helm/aws-operator/templates/_helpers.tpl

```diff
@@ -0,0 +1,34 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "aws-operator.name" -}}
+{{- .Chart.Name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "aws-operator.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Common labels
+*/}}
+{{- define "aws-operator.labels" -}}
+helm.sh/chart: {{ include "aws-operator.chart" . | quote }}
+{{ include "aws-operator.selectorLabels" . }}
```

Comment:
I'm thinking now we should add `app` here. People will freak out when commands using `app` as a selector in their shell history don't work anymore.

Reply:
Done

Reply:
What would be the process to deprecate it then? Or do we maintain it indefinitely?

Reply:
I'm not too sure about this, I don't like keeping cruft. I understand I may be in the minority here but I'd like to have a decision on what do we do with it going forward:

1. maintain it indefinitely, i.e. touch, move on
2. keep it for now, mark as deprecated in fmt, allow folks to adjust, remove some other time
3. bite the bullet, get it cleaned up now, get on with life

Reply:
I'd like to simply keep it to not cause disruption in people workflows. Feel free to drive a removal of this thing and I will support you. But that would require some prior discussions and agreements. I wouldn't like to just silently introduce that as a side-effect of som task because I guarantee you it will annoy people for a while.

Reply:
Check this out https://github.com/giantswarm/giantswarm/issues/9493

Reply:
Done, and I updated fmt PR.